guardian/editorial-tools-integration-tests
# Convert all remaining JS helper files to Typescript!

## What does this change?

It turns out that Cypress can handle any Typescript files passed to it, and Typescript is great, so this PR converts all of the remaining JS files to TS so we can benefit from proper typing and all of the other wonderful TS goodness.

This PR mainly converts the filenames and does a bit of Typescriptifying in order for the files to compile properly.

## How can we measure success?

The test runners continue to work as normal.

## How to test?

Locally, run `./scripts/run.sh <STAGE>` to try out the test runner and see that it runs the tests successfully.

## Do the relevant integration tests still pass?

[X] Tested against ALL CODE

## Images
